### PR TITLE
API docs

### LINKED ISSUE #4: API docs
We closed #2 without them. Need to be completed.

### PR DESCRIPTION
**Pull request by szymonkups**
Fixes #4.

### CODE REVIEW

**Comment by pomek** [comment]
According to https://github.com/ckeditor/ckeditor5/issues/107#issuecomment-226725130, I have to put label `r-`

File: src/headingscommand.js [hunk 1]
```diff
@@ -33,13 +58,16 @@ export default class HeadingsCommand extends Command {
...
+	 * @type {headings.HeadingsFormat}
 	 */
 	get defaultFormat() {
 		// See https://github.com/ckeditor/ckeditor5/issues/98.
 		return this._getFormatById( 'paragraph' );
 	}
 
+	/**
+	 * @inheritDoc
+	 */
```
Comment (Reinmar) [inline src/headingscommand.js]: You'd like to document the formatId param here.

File: src/headingsengine.js [hunk 2]
```diff
@@ -18,11 +18,25 @@ const formats = [
 	{ id: 'heading3', viewElement: 'h4', label: 'Heading 3' }
 ];
 
+/**
+ * The headings feature. Handles switching between block formats - different headings and paragraph.
+ * This class represents part of the feature that can be used without any UI - for example inside
```
Comment (Reinmar) [inline src/headingsengine.js]: I wouldn't write this so explicitly here that engine part can be used in Node.js. I'd keep it general that: "This feature represents the engine part of the Headings feature."

File: src/headings.js [hunk 3]
```diff
@@ -12,11 +12,23 @@ import ListDropdownController from '../ui/dropdown/list/listdropdown.js';
 import ListDropdownView from '../ui/dropdown/list/listdropdownview.js';
 import Collection from '../utils/collection.js';
 
+/**
+ * The headings feature. Handles switching between different block types.
```
Comment (Reinmar) [inline src/headings.js]: Those "different block types" needs better explanation. Or even: "The headings feature. Introduces a "headings" drop-down and a command which allow switching paragraphs into headings."

**Pull request merged**